Jetcaster/mobile/src/main/java/com/example/jetcaster/ui/MainActivity.kt

```diff
@@ -23,14 +23,15 @@ import androidx.activity.enableEdgeToEdge
 import com.example.jetcaster.ui.theme.JetcasterTheme
 import com.google.accompanist.adaptive.calculateDisplayFeatures
 import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.runBlocking
 
 @AndroidEntryPoint
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
         enableEdgeToEdge()
-
+        runBlocking { com.example.jetcaster.glancewidget.updateWidgetPreview(this@MainActivity) }
```

Comment:
This seems dangerous? How does StrictMode react to it? Why does it need to be runBlocking?

Reply:
updateWidgetPreview is suspend, I'm up for any other one liner, but runBlocking has always been the easiest/fastest. 

How do I check strict mode